test/prevent-errortype.js

```diff
@@ -0,0 +1,32 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/prevent-errortype';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const errors = [{ruleId: 'prevent-errortype'}];
+
+const header = 'const test = require(\'ava\');\n';
+
+ruleTester.run('prevent-errortype', rule, {
```

Comment:
Needs some more tests. For example, one where the second argument is `true`, `string`, and `object`.

Reply:
added.